### ISSUE TITLE
gmic build failed

### ISSUE THREAD

**Issue by LoCrealloc** [NixOS/nixpkgs#212052]
### Steps To Reproduce
1. Open gmic in a nix shell using the unstable channel

### Build log
https://pastebin.com/SwdL7gkE

### Notify maintainers
@ryantm @jtojnar-

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.89, NixOS, 23.05 (Stoat), 23.05pre445203.5ed48194335`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.12.0`
 - channels(root): `"nixos"`
 - channels(loc): `"home-manager-22.11.tar.gz"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by jtojnar** [comment]
Will be fixed by #211600

**Comment by tobiasBora** [comment]
Note that it makes darktable fail to build.
